launcher/ui/pages/instance/OtherLogsPage.cpp

```diff
@@ -173,7 +193,34 @@ void OtherLogsPage::on_btnReload_clicked()
             showTooBig();
             return;
         }
-        setPlainText(content);
+
```

Comment:
I believe for the sake of performance it would be best to set the model to nullptr before adding lines and settings it back to m_proxy after adding them.

This is because https://github.com/PrismLauncher/PrismLauncher/pull/2112 greatly improved the performance of inserting multiple lines at once and unfortunately the model only supports one at a time. 

Hopefully this limitation is fixed in the future as it would probably improve performance when the log is moving quickly, but for now if the view doesn't see any of these lines until they are all inserted performance is better.